docs/reference/commandline/cp.md

```diff
@@ -11,12 +11,12 @@ weight=1
 
 # cp
 
-Copy files/folders between a container and the local filesystem.
+    Usage: docker cp [OPTIONS] CONTAINER:PATH LOCALPATH|-
+           docker cp [OPTIONS] LOCALPATH|- CONTAINER:PATH
 
-    Usage:  docker cp [options] CONTAINER:PATH LOCALPATH|-
-            docker cp [options] LOCALPATH|- CONTAINER:PATH
+    Copy files/folders between a container and the local filesystem
 
-    --help  Print usage statement
+      --help=false        Print usage
```

Comment:
have you checked the generated output here? Changing indentation in Markdown can be tricky (not sure if it's troublesome in the man-pages here though)